gc/scheduler/scheduler.go

```diff
@@ -28,6 +29,61 @@ import (
 	"github.com/pkg/errors"
 )
 
+var pluginID = "scheduler"
+var commentedSchedulerConfigTemplate = template.Must(template.New(pluginID).Parse(`
+# The "plugins.scheduler" configures the garbage collection policies.
+[plugins.scheduler]
+
+  # pause_threshold represents the maximum amount of time garbage
+  # collection should be scheduled based on the average pause time.
+  # For example, a value of 0.02 means that scheduled garbage collection
+  # pauses should present at most 2% of real time,
+  # or 20ms of every second.
+  #
+  # A maximum value of .5 is enforced to prevent over scheduling of the
+  # garbage collector, trigger options are available to run in a more
+  # predictable time frame after mutation.
+  #
+  # Default is 0.02
```

Comment:
I think we don't need default values here

Reply:
@AkihiroSuda Done.